### DISCUSSION TITLE
Watched / rating section

### DISCUSSION THREAD

**Discussion by Drallas** [sct/overseerr#3307] [Feature Requests]
It would be nice to have a section to rate movies and update them too at services like TMDB / IMDB / TVDB. 

Even better, pull my watched items and stars from them into my personal profile, displaying my rating on the items in Overseerr.

**Comment by brianjmurrell** [comment]
Even just being able to mark watched so that they are removed from lists.  One of the things I find most annoying about, say, Netflix, is that you cannot filter out things you have watched from their lists so you are constantly sifting through their lists looking for things you have not yet watched.  Overseerr seems to have that same problem unless there is a feature I am missing somewhere.
